README.md

```diff
@@ -72,7 +72,11 @@ Install dependencies. For Debian-based distros:
 
 For Arch-based distros:
 
-	$ sudo pacman -S gtk2 curl freetype libpng libjpeg sdl2 sdl2_gfx sdl2_image sdl2_mixer sdl2_ttf
+	$ sudo pacman -S gtk2 curl freetype libpng libjpeg sdl2 sdl2_gfx sdl2_image-devel sdl2_mixer sdl2_ttf
+
+For Fedora:
+	$ sudo dnf groupinstall "Development Tools"
```

Comment:
Add newline between "For Fedora" and the first command, otherwise the markdown formatting messes up